tests/docs/smoke-all/confluence/.gitignore

```diff
@@ -1,4 +1,6 @@
 _publish.yml
 *.xml
+5815-confluence-links-to-file-attachments-not-supported_files
+*.html
```

Comment:
Do you need to add this specifically for this example ? 

I don't think we add all the smoke-all folders into this gitignore. So I am wondering what is specific about this one ? 

The html and resources files should be cleaned up after the smoke test